bazel/repositories.bzl

```diff
@@ -596,7 +596,14 @@ def _com_googlesource_quiche():
     )
 
 def _com_github_grpc_grpc():
-    _repository_impl("com_github_grpc_grpc")
+    _repository_impl(
+        "com_github_grpc_grpc",
+        patches = [
```

Comment:
Can you file an issue and mention it as a TODO comment here to remove the patches once a new gRPC is released? If the patch is based on a gRPC master PR, link there in the comment too.

Reply:
https://github.com/envoyproxy/envoy/issues/7863 . Also added a comment.